### ISSUE TITLE
Control ROM volume from VPX / 'F12' (L/R channels)

### ISSUE THREAD

**Issue by jueank** [vpinball/vpinball#1275]
tl;dr Add a volume option to the 'F12' menu for the L/R channel for 'ROM volume'. (Because in 7.1 setups, ROM volume is coming from these 2 channels). And store this with the other settings per table. This makes managing table volumes across many table much simpler!

Long version:
In my search for simplifying the process of controlling pinball audio / managing hundreds of table volume settings. I try to get rid of as many 'knobs' to turn as possible. The challenge is to balance out the volumes of the ROM audio and the table audio and have a volume per table that is consistent across multiple tables.
In order to achieve that, currently the users have to use helper programs, such as PinVol (per table audio, including ROM volume). But that only makes the process of handling tables + audio more complex, because some settings are redundant to VPX per table settings. Having many ways of influencing audio volume is also a rather obscure thing, because the values are not shown on screen (or only in the moment when they are changed). So the user does not really know what is happening behind the scenes.
Volume is influenced by
- Hardware volume (set on your AMPs. Needs to be balanced between backbox speakers (ROM) + cab speakers (table))
- Windows global volume
- PinVol per table volume (overall)
- VPX per table volume (music + sounds)
- PinMAME ROM volume (usually avoided to be set directly, because that is a pain to do)

With the introduction of the 'F12' menu, the users have a convenient way to set Music + sound volume **per table** during play. Also the values can be seen on screen. This draws helper programs like PinVol almost redundant. Almost, because there is no way how to set ROM volume per table in a convenient way and store it with the table. Unfortunately, especially the ROM volumes are very different and some tables from the 80s have very loud beep sounds that are nerve-wrecking when heard for longer.

An ideal way would be to add a ROM volume option to the F12 menu and have an integration with PinMAME in order to set ROM volume when the table launches.
OR - if a complex integration with PinMAME is not feasible, let VPX control the L/R speaker volume (and call that ROM volume)

**Comment by rudil24** [comment]
yes! also those of us that used F12 old rom volume menu to attenuate rom volume on previous VPX versions like 10.6  (before the "new" F12 menu came out), our ROM attenuate values are seemingly locked forever at whatever we put in that menu at that time (0 to -32db), am i right?

**Comment by jueank** [comment]
With the plugin system - and hopefully being able to set VPinMAME ROM volume at runtime, plus the other sound options in InGameUI, this becomes obsolete soon.
